test/hexpm/web/controllers/api/package_download_controller_test.exs

```diff
@@ -0,0 +1,145 @@
+defmodule Hexpm.Web.API.PackageDownloadControllerTest do
+  use Hexpm.ConnCase, async: true
+
+  setup do
+    user = insert(:user)
+    unauthorized_user = insert(:user)
+    organization = insert(:organization)
+
+    package1 =
+      insert(
+        :package,
+        name: "Hexpm.Web.API.PackageControllerTest",
+        inserted_at: ~N[2030-01-01 00:00:00]
+      )
+
+    package2 = insert(:package, updated_at: ~N[2030-01-01 00:00:00])
+
+    package3 =
+      insert(:package, organization_id: organization.id, updated_at: ~N[2030-01-01 00:00:00])
+
+    package4 = insert(:package)
```

Comment:
Note for myself: I could cleanup and avoid creating resources that will not be used